src/Razor/test/Microsoft.NET.Sdk.Razor.Test/Microsoft.NET.Sdk.Razor.Test.csproj

```diff
@@ -92,7 +92,7 @@ namespace Microsoft.AspNetCore.Razor.Design.IntegrationTests
 
   </Target>
 
-  <Target Name="RestoreTestProjects" BeforeTargets="Restore">
+  <Target Name="RestoreTestProjects" BeforeTargets="Restore;Build">
```

Comment:
Why was this needed? Doesn't Restore run before Build anyway?

Reply:
I have restore turned off in VS. But we absolutely need for restore to happen for the tests to succeed now.